src/main/java/cn/nukkit/command/defaults/GameruleCommand.java

```diff
@@ -16,9 +15,9 @@
 public class GameruleCommand extends VanillaCommand {
     private final GameRuleRegistry registry;
 
-    public GameruleCommand(String name, Server server) {
+    public GameruleCommand(String name) {
         super(name, "%nukkit.command.gamerule.description", "%nukkit.command.gamerule.usage");
-        this.registry = server.getGameRuleRegistry();
+        this.registry = GameRuleRegistry.get();
```

Comment:
Instead, this should be assigned at the variable declaration, not in the constructor and be static.

Reply:
done, and PR updated